proto/api.proto

```diff
@@ -409,6 +420,13 @@ message GetOrderbookRequest {
   uint32 limit = 2 [(grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {description: "Use 0 for no limits"}];
 }
 
+message GetOrderbookRequestWithInfo {
+  string market = 1;
+  uint32 limit = 2 [(grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {description: "Use 0 for no limits"}];
+  uint32 height = 3;
+  bool isNodeHealthy = 4;
```

Comment:
Don't think the above 2 are needed. In fact you can reuse the GetOrderbookRequest message for your endpoint without changes